From 65dbd552f86f6178382806d265c1c61d420ecb56 Mon Sep 17 00:00:00 2001
From: wwidjaja <w.widjaja@intel.com>
Date: Wed, 12 Jun 2019 17:05:57 +0800
Subject: [PATCH] DRVLIB_SW-2354 - PRX300 bootcore up notification

  fix no mps/icc message to indicate to IAP that bootcore is ready
  when PRX300 SoC.
---
 drivers/char/ltq_icc/drv_icc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/char/ltq_icc/drv_icc.c b/drivers/char/ltq_icc/drv_icc.c
index 3a192ec51a1e..bb6024bba751 100644
--- a/drivers/char/ltq_icc/drv_icc.c
+++ b/drivers/char/ltq_icc/drv_icc.c
@@ -384,7 +384,7 @@ long icc_ioctl (struct file *file_p,uint32_t nCmd, unsigned long arg){
 		 			/*mark that the device is opened in global structure
 					to avoid further open of the device*/
 		 			iccdev[num].Installed=1;
-#ifdef CONFIG_SOC_GRX500_BOOTCORE
+#if defined(CONFIG_SOC_GRX500_BOOTCORE) || defined(CONFIG_SOC_PRX300_BOOTCORE)
 					/*Intimate to IAP that bootcore is ready if IA gets registered*/
 					if(num == IA)
 					{
